### PR TITLE
Fixed react-json-tree - Broken import statement in d.ts #1087

### PR DESCRIPTION
**Pull request by NaincyKumariKnoldus**
Hey ! I have fixed react-json-tree - Broken import statement in d.ts. Please review my PR.
If still any changes needed do inform me I will do that.
Thanks!

#1087

### CODE REVIEW

File: packages/react-json-tree/src/index.tsx [hunk 1]
```diff
@@ -7,12 +7,12 @@ import React from 'react';
...
   invertTheme,
-  type StylingConfig,
-  type StylingFunction,
-  type StylingValue,
-  type Theme,
+  StylingConfig,
+  StylingFunction,
+  StylingValue,
+  Theme,
 } from 'react-base16-styling';
```
Comment (Methuselah96) [inline packages/react-json-tree/src/index.tsx:16]: You'll need to split this into two separate import statements, one for the types and one for `invertTheme` which is not a type.

**Review by Methuselah96 — CHANGES_REQUESTED** [review]

File: packages/react-json-tree/src/index.tsx (hunk 1 above)
Comment (NaincyKumariKnoldus) [reply packages/react-json-tree/src/index.tsx]: Hey @Methuselah96 I have made the requested changes. Review it now.
Thanks!

File: packages/react-json-tree/src/index.tsx [hunk 2]
```diff
@@ -7,12 +7,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import JSONNode from './JSONNode';
 import createStylingFromTheme from './createStylingFromTheme';
-import {
-  invertTheme,
-  type StylingConfig,
-  type StylingFunction,
-  type StylingValue,
-  type Theme,
+import invertTheme from 'react-base16-styling';
```
Comment (Methuselah96) [inline packages/react-json-tree/src/index.tsx:10]:
```suggestion
import { invertTheme } from 'react-base16-styling';
```

**Review by Methuselah96 — CHANGES_REQUESTED** [review]

File: packages/react-json-tree/src/index.tsx (hunk 2 above)
Comment (NaincyKumariKnoldus) [reply packages/react-json-tree/src/index.tsx]: Hey @Methuselah96 made the changes. Review it now.

**Comment by Methuselah96** [comment]
Thanks, can you add a changeset?

**Comment by edzis** [comment]
@NaincyKumariKnoldus could you please address the last remaining comment?

**Comment by NaincyKumariKnoldus** [comment]
> @NaincyKumariKnoldus could you please address the last remaining comment?

Hey @edzis actually i dont know how to add a changeset? Can you do this please.

**Review by Methuselah96 — APPROVED** [review]

**Pull request merged**